components/specials/diary/ArrowIcon.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// TODO: to consider moving into babajka/assets repo.
+
+const ArrowIcon = ({ size, direction }) => (
+  <svg
+    width={size}
+    height={size}
+    viewBox="0 0 35 16"
+    fill="none"
+    style={{ transform: direction === 'right' ? 'rotate(180deg)' : '' }}
```

Comment:
👍

Reply:
for future: svg also have `transform` operation, but it does not work in safari and iOS safari. but CSS-transform does work :)